tests/run/test_Saver.py

```diff
@@ -0,0 +1,113 @@
+import copy
+from tempfile import TemporaryDirectory
+
+import pytest
+import torch
+from hypothesis import given
+from hypothesis import settings
+from hypothesis import strategies as st
+from myrtlespeech.model.seq_to_seq import SeqToSeq
+from myrtlespeech.protos import task_config_pb2
+from myrtlespeech.run.callbacks.callback import CallbackHandler
+from myrtlespeech.run.run import Saver
+from myrtlespeech.run.train import fit
+
+from tests.builders.test_task_config import build_and_check_task_config
+from tests.protos.test_task_config import task_configs
+from tests.utils.utils import check_state_dicts_match
+
+# Utilities -------------------------------------------------------------------
+
+
+def change_seq_to_seq_state_dict(seq_to_seq: SeqToSeq):
```

Comment:
Name is a little opaque, could make clearer if you want to update or have any suggestions (I can't think of any, naming is hard!). Name change aside, perhaps adding an underscore to signify in-place and updating the top level description? 

```suggestion
def change_seq_to_seq_state_dict_(seq_to_seq: SeqToSeq) -> None:
    """Modifies all values in ``seq_to_seq`` such that they are different"""
```

Reply:
Changed to `alter_seq2seq_attributes_`?